src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -173,6 +168,8 @@ public ProcessedArgs(
         AggregateProperties.TryGetValue(SonarProperties.Region, out var region);
         AggregateProperties.TryGetValue(SonarProperties.ApiBaseUrl, out var apiBaseUrl);
 
+        TelemetryUtils.AddTelemetry(logger, AggregateProperties);
```

Comment:
I would move this directly after the `new AggregatePropertiesProvider` a few lines above.